libs/components/core/src/lib/modules/affix/dom-utils.ts

```diff
@@ -65,12 +65,14 @@ export function getOverflowParents(child: HTMLElement): HTMLElement[] {
       overflowY === 'hidden' ||
       overflowY === 'scroll'
     ) {
-      results.push(parentElement);
+      results.unshift(parentElement);
     }
 
     parentElement = parentElement.parentNode;
   }
 
+  results.unshift(bodyElement);
```

Comment:
Using `Array.unshift` is generally discouraged due to performance reasons. What's the purpose of this change?